.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+defaults: &defaults
+  docker:
+    - image: circleci/node:8
+  working_directory: ~/nodejs
+
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-yarn-cache-{{ checksum "yarn.lock" }}
+          - v1-yarn-cache-
+      - run:
+          name: Install greenkeeper-lockfile-update
+          command: yarn global add greenkeeper-lockfile@1 --prefix ~/.yarn
+      - run:
+          name: Install dependencies
+          command: yarn install --pure-lockfile
+      - save_cache:
+          key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+      - run:
+          name: Update greenkeeper-lockfile
+          command: ~/.yarn/bin/greenkeeper-lockfile-update
+      - run:
+          name: Upload greenkeeper-lockfile
```

Comment:
I don't think this has to run at the complete end of a chain as mentioned in their docs as dependencies won't change after the install.